lnxrouter

```diff
@@ -179,8 +179,9 @@ define_global_variables(){
     WIFI_IFACE=
     CHANNEL=default 
     WPA_VERSION=2
-    MAC_FILTER=0
+    MAC_FILTER=3  # 3 is not valid
```

Comment:
Leave it blank (or use -1 or -99) is better. hostapd may add 3 as a valid value in the future